ocs_ci/ocs/elasticsearch.py

```diff
@@ -245,3 +306,38 @@ def _copy(self, es):
             for doc in result["hits"]["hits"]:
                 log.debug(f"Going to write : {doc}")
                 es.index(index=ind, doc_type="_doc", body=doc["_source"])
+
+    def dump(self, target_path):
+        """
+        Dump All data from the internal ES server to .tgz file.
+
+        Args:
+            target_path (str): the path where the results file will be copy into
+
+        Return:
+            bool: True if the dump operation succeed and return the results data to the host
+                  otherwise False
+        """
+
+        log.info("dumping data from ES server to .tgz file")
```

Comment:
Same as before - in case you agree, please start from capital D , "Dumping" instead of "dumping"

Reply:
done